Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM gradle:jdk11-alpine AS builder
+
+COPY *.gradle.kts gradle.properties manifest.txt /app/
+COPY gradle/ /app/src/
+COPY src/ /app/src/
+COPY skin/ /app/skin/
+COPY stdlib/ /app/stdlib/
+COPY svg/ /app/svg/
+COPY test/ /app/test/
+COPY themes/ /app/themes/
+
+WORKDIR /app
+RUN gradle --no-daemon clean build -x javaDoc -PjavacRelease=11 -x test
+
+RUN find /app -name '*.jar'
+
+################################
+
+FROM openjdk:11
```

Comment:
Is there a reason for not using `11-jre-slim-buster` here?

(The reason for asking is that the created Docker image is half the size with Slim...)